### PR TITLE
Log actual originalUrl after workspace proxy errors

### PR DESCRIPTION
**Pull request by nwalters512**

### CODE REVIEW

File: middlewares/selectAndValidateWorkspace.sql [hunk 1]
```diff
@@ -1,8 +0,0 @@
--- BLOCK select_workspace_hostname
```
Comment (nwalters512) [inline middlewares/selectAndValidateWorkspace.sql:1]: This file should have been removed with #6260.

File: server.js [hunk 2]
```diff
@@ -338,8 +338,13 @@ module.exports.initExpress = function () {
   app.use('/pl/workspace/:workspace_id/container', [
     cookieParser(),
     (req, res, next) => {
-      // Needed for workspaceAuthRouter and `selectAndValidateWorkspace` middleware.
+      // Needed for workspaceAuthRouter.
       res.locals.workspace_id = req.params.workspace_id;
+
+      // Needed for logs; this will retain the original URL if we end up
+      // rewriting the URL in `pathRewrite` above.
+      req.originalUrl = req.url;
```
Comment (mwest1066) [inline server.js:346]: I thought express automatically set `req.originalUrl` for us? (see [the docs](https://expressjs.com/en/api.html#req.originalUrl))

File: server.js [hunk 3]
```diff
@@ -304,7 +304,7 @@ module.exports.initExpress = function () {
       logger.error(`Error proxying workspace request: ${err}`, {
         err,
         url: req.url,
-        originalUrl: req.url,
+        originalUrl: req.originalUrl,
```
Comment (mwest1066) [inline server.js:307]: I just read this line in the code and somehow I never saw this!!!

**Review by mwest1066 — APPROVED** [review]

File: server.js (hunk 2 above)
Comment (nwalters512) [reply server.js:346]: Ah, true! Good catch.

Comment (nwalters512) [reply server.js]: Resolved in 2fe82a9.

**Pull request merged**